features/step_definitions/user_steps.rb

```diff
@@ -205,3 +206,16 @@ def with_scope(locator)
 Then(/^I have updated the vendor home$/) do
   pending # Write code here that turns the phrase above into concrete actions
 end
+
```

Comment:
The 2 steps below are not used in any cucumber test (yet).  So maybe I should remove them?